PKGBUILD

```diff
@@ -4,15 +4,14 @@
 # Contributor: demostanis worlds <demostanis@protonmail.com>
 
 pkgname=archinstall
-pkgver=2.6.0
+pkgver=2.7.2
 pkgrel=1
 pkgdesc="Just another guided/automated Arch Linux installer with a twist"
 arch=(any)
 url="https://github.com/archlinux/archinstall"
-license=(GPL3)
+license=(GPL-3.0-or-later)
```

Comment:
According to the project information it is `GPL-3.0-only` though?

Reply:
hmm, let me double check. if that's the case I'll change.